extras/python/fogbench/__main__.py

```diff
@@ -213,6 +210,9 @@ def read_out_file(_file=None, _keep=False, _iterations=1, _interval=0, send_to='
      URI "/other/sensor-values".
 
     """
+    from aiocoap import Context, Message, numbers
```

Comment:
@rajeshkumar426 do we need the `numbers` import?

Reply:
We need `numbers` import for `POST`.